processing/src/main/java/org/apache/druid/segment/column/DruidPredicateIndex.java

```diff
@@ -36,4 +36,12 @@
    */
   @Nullable
   BitmapColumnIndex forPredicate(DruidPredicateFactory matcherFactory);
+
+  static boolean checkSkipThreshold(@Nullable ColumnConfig columnConfig, int numRowsToScan, int dictionaryCardinality)
```

Comment:
`shouldSkip`?  `checkSkipThreshold` doesn't really tell me how I'm supposed to interpret `true/false`.  Does `true` mean that it exceeded the threshold and therefore should be skipped?  Or does it mean that it did not exceed the threshold and should be used?